website/pages/docs/job-specification/expose.mdx

```diff
@@ -26,46 +26,100 @@ Service [check](https://nomadproject.io/docs/job-specification/service/#check-pa
 configurations can use their [expose](/docs/job-specification/service#expose)
 parameter to automatically generate expose path configurations for HTTP and gRPC checks.
 
+```hcl
+job "expose-check-example" {
+  datacenters = ["dc1"]
+
+  group "api" {
+    network {
+      mode = "bridge"
+    }
+
+    service {
+      name = "count-api"
+      port = "9001"
+
+      connect {
+        sidecar_service {}
+      }
+
+      check {
+        expose   = true
+        name     = "api-health"
+        type     = "http"
+        path     = "/health"
+        interval = "10s"
+        timeout  = "3s"
+      }
+    }
+
+    task "web" {
+      driver = "docker"
+
+      config {
+        image = "hashicorpnomad/counter-api:v2"
+      }
+    }
+  }
+}
+```
+
+For uses other than Consul service checks, use the `expose` configuration in the
```

Comment:
I feel like this text is suggesting that there's some non-check related use case but then the example is just another health check. Might someone use `expose` for allowing ingress traffic to the workload? If so, maybe we could call that out specifically?

Reply:
I don't think ingressing traffic is the target usecase. Usually you would expose things like a prometheus endpoint so that the host agent could scrape metrics for example.